backend/api/tests/test_models.py

```
@@ -0,0 +1,156 @@
+    """Test the string representation of the Video model."""
+    user = User.objects.create_user(email='uploader3@example.com', firebase_uid='uploaderuid3')
+    video = Video.objects.create(
+        title='Video Str Test',
+        video_url='http://example.com/video3.mp4',
+        uploader=user
+    )
+    assert str(video) == 'Video Str Test'
+
+# --- Profile Model Tests (Basic Existence) ---
+
+def test_company_profile_creation():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_company_profile_creation`

Add return type annotation: `None`

(ANN201)